start_server.py

```
@@ -0,0 +1,93 @@
+print("2. Detect new filings in real-time")
+print("3. Display filing alerts when detected")
+print("4. Save detected filings to detected_filings.json")
+print("=" * 60)
+print("\nPress Ctrl+C to stop the server\n")
+
+# Clear previous filings
+try:
+    with open("detected_filings.json", "w") as f:
+        json.dump([], f)
+    print("✅ Cleared previous filing detections")
+except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)